vignettes/pull-nhsn.Rmd

```diff
@@ -21,7 +21,7 @@ We will use the [Socrata Open Data API (SODA)](https://dev.socrata.com/) API end
 
 
 ## API Key
-First, you'll want to go to `data.cdc.gov` and request an API token. This is not strictly required, but it is generally considered polite, and it will speed up your data requests (the polite get served first!).
+First, you'll want to go to [data.cdc.gov](https://data.cdc.gov/) and request an API token. This is not strictly required, but it is generally considered polite, and it will speed up your data requests (the polite get served first!).
```

Comment:
```suggestion
First, you'll want to go to [`data.cdc.gov`](https://data.cdc.gov/) and request an API token. This is not strictly required, but it is generally considered polite, and it will speed up your data requests (the polite get served first!).
```